### DISCUSSION TITLE
File is NOT spell checked because it is not in the workspace.

### DISCUSSION THREAD

**Discussion by VisualGPS** [streetsidesoftware/vscode-spell-checker#1308] [Q&A] [answered]
This is new. Do we need to have a "workspace" to spell check? This was on a mark-down README.md file. This once was working or I did I cause this somehow.

**Comment by Jason3S** [comment, marked as answer]
@VisualGPS,

Did you try the `spell check only workspace files` setting?

![image](https://user-images.githubusercontent.com/3740137/133642239-fdbfa77e-88f4-46c7-ba0c-c47f906cc9a1.png)

**Reply by VisualGPS** [reply]
TY: It took a bit of finding and it was checked -- never even been to C Spell settings before and not sure why it was checked! Fixed now.
